src/Microsoft.Fx.Portability.Reports.Html/Resources/_PortabilityReport.cshtml

```diff
@@ -144,7 +144,19 @@
         detailColumnHeaders.Add(LocalizedStrings.TargetTypeHeader);
         detailColumnHeaders.AddRange(Model.TargetHeaders);
         detailColumnHeaders.Add(LocalizedStrings.RecommendedChanges);
-        <table>
+
+        <form>
+            Hide columns: 
+            @* It is not allowed to hide the first (Api name) and the last (Recommended Changes) columns. *@
```

Comment:
We don't allow as it wouldn't make sense to hide the first column.  Maybe allow hiding the last?